idd/Energy+.idd.in

```diff
@@ -34502,6 +34502,4630 @@ ZoneHVAC:EvaporativeCoolerUnit,
         \type object-list
         \object-list DesignSpecificationZoneHVACSizingName
 
+ZoneHVAC:HybridUnitaryHVAC,		
```

Comment:
Although the C++ code does have tabs, the IDD does not.  Convert them to spaces.